### PR TITLE
New Gedit Cheat Sheet

### PR DESCRIPTION
**Pull request by souravbadami**
Cheat sheet to display gedit keyboard shortcuts.
- [x] New Instant Answer
    - [x] Cheat Sheets: **`Gedit Cheat Sheet`**

Instant Answer Page: https://duck.co/ia/view/gedit_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@souravbadamiThanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=gedit cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/gedit_cheat_sheet_mobile.png?2KpUGa6zOe)

File: share/goodie/cheat_sheets/json/gedit.json [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+            "val":"Goto line"
+        }],
+        "Tools": [{
+            "key":"[Shift] [F7]",
+            "val":"Check spelling"
+          }, {
+            "key":"[Alt] [F12]",
+            "val":"Remove trailing spaces (with plugin)"
+          }, {
+            "key":"[F8]",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/gedit.json]: For single keys you don't need to use the `[]` symbol.

Please change this to `"key":"F8",`

File: share/goodie/cheat_sheets/json/gedit.json [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+            "val":"Run 'make' in the current directory (with plugin)"
+          }, {
+            "key":"[Ctrl] [Shift] [D]",
+            "val":"Directory listing (with plugin)"
+        }],
+        "User Help": [{
+            "key":"[F1]",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/gedit.json]: Also here,,,

File: share/goodie/cheat_sheets/json/gedit.json [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+            "val":"Toggle case of the selected text"
+        }],
+        "Panes": [{
+            "key":"[F9]",
+            "val":"Show / hide the side pane"
+          }, {
+            "key":"[Ctrl] [F9]",
+            "val":"Show / hide the bottom pane"
+          }, {
+            "key":"[F10]",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/gedit.json]: Here

**Comment by jdorweiler** [comment]
:+1

**Pull request merged**